client-ui/src/app/components/lineage/lineage-details/operation-properties-details/extra-properties/extra-properties-expression.component.ts

```diff
@@ -25,10 +25,10 @@ export class ExtraPropertiesExpressionComponent {
 
   @Input() properties: OperationProperty.ExtraPropertyValueExpression[]
 
-  @Output() valueClicked$ = new EventEmitter<{ property: OperationProperty.ExtraPropertyValueExpression }>()
+  @Output() expressionClicked$ = new EventEmitter<OperationProperty.ExpressionValue>()
```

Comment:
Why did you refactor EventEmitter payload structure?

Reply:
Using an object as EventEmitter payload should be preferred over pure value, even if it is the only one value there so far.

By doing so you are prepared for easy payload extension and also aby calling object property in a proper way you are doing life easier with working with payload in other places.

Reply:
Good point for readability in the usage place, e.g. `$event.name` versus `$event.property.name`. Thanks.

Reply:
Although I like the suggestion in general, in this particular case however, I don't think it adds any benefits. The payload itself is already a wrapper with quite opaque semantics:
```typescript
export type ExpressionValue = {
    value: string
    rawValue: Record<string, any>
  }
``` 
It I create another wrapper on a wrapper that would look strange. I even struggle to create a meaningful name for it. E.g.
```typescript
// definition 
new EventEmitter<{ value: OperationProperty.ExpressionValue }>
// usage
"$event.value.value" // Doh...
```

Reply:
it will be even worse if you would call as X, like so:

`
new EventEmitter<{ x: OperationProperty.ExpressionValue }>
`

but my point is to call it in a way it will help you to understand what kind of value are you emitting here, e.g

`
new EventEmitter<{ expressionValue: OperationProperty.ExpressionValue }>
`

Reply:
it's still `...value.value`. But OK, let's do it.